index.md

```diff
@@ -28,4 +28,4 @@ Live demo is available on [http://bootzooka.softwaremill.com](http://bootzooka.s
 
 ## License
 
-Project is licensed under [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html) which means you can freely use any part of the project.
```

Comment:
Hmm... I didn't change the line. Maybe be some side effect from GH online editing. Sorry for didn't noticing.

I will update the PR.